test/e2e/tests/metrics_test_eraser/eraser_test.go

```diff
@@ -23,7 +23,7 @@ func TestMetrics(t *testing.T) {
 	metrics := features.New("Images_removed_run_total metric should report 1").
 		Assess("Alpine image is removed", func(ctx context.Context, t *testing.T, cfg *envconf.Config) context.Context {
 			// deploy imagelist config
-			if err := util.DeployEraserConfig(cfg.KubeconfigFile(), util.TestNamespace, "../../test-data", "imagelist_alpine.yaml"); err != nil {
+			if err := util.DeployEraserConfig(cfg.KubeconfigFile(), util.EraserNamespace, "../../test-data", "imagelist_alpine.yaml"); err != nil {
```

Comment:
Why these changes?

Reply:
wanted to get logs for the metrics tests, but the other e2e tests had been using util.EraserNamespace instead of TestNamespace and that is what was used to get the logs. I can put up a separate PR to fix that to consistently use TestNamespace instead!

Reply:
I'll put up a PR for that and you can rebase this PR on top of that. Best to keep things orderly when possible. All the tests are using `util.TestNamespace` except for the `collector_pipeline` test because it's not deployed via helm.

Reply:
In all the `eraser_test.go` files, we should be be using `cfg.Namespace()` for all namespace references. That way, we can't get ourselves into trouble with having conflicting namespaces, it's just set once in `main_test.go` and everything else refers to that.

Reply:
They create the TestNamespace, but still use EraserNamespace when deploying Eraser. The get logs methods will need to be changed too to use TestNamespace instead or take namespace as an argument.

Reply:
yeah, they need to take a namespace as an argument.